IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -156,7 +156,15 @@ - (BOOL)setItem:(MSIDToken *)item
     // Backward compatibility with ADAL.
     [query setObject:[s_libraryString dataUsingEncoding:NSUTF8StringEncoding] forKey:(id)kSecAttrGeneric];
     // Backward compatibility with MSAL.
-    [query setObject:[NSNumber numberWithUnsignedInt:item.tokenType] forKey:(id)kSecAttrType];
+    if (item.tokenType == MSIDTokenTypeAccessToken)
+    {
+        [query setObject:[NSNumber numberWithUnsignedInt:'acTk'] forKey:(id)kSecAttrType];
```

Comment:
nit: can we have an enum to hold acTk and rfTk?

Reply:
We used to have it, but the problem is that by default enum is initialized with "0", when 'acTk' != 0. As a result enum has invalid value. We can assign default value of enum in initializer, but since 'acTk' and 'rfTk' are used only in MSAL, I think it is better to hide such platform specific details inside the class.